### ISSUE TITLE
connections - Wireless Links

### ISSUE THREAD

**Issue by baendres** [iDebugAll/nextbox-ui-plugin#93]
connections created between two wireless devices under the connections/wireless links section do not show up as connected in the topology view.  Is there a way to make these devices show up as connected with a different color or line type?

**Comment by snatch2013** [comment]
yes, the same problem. netbox added the support for wireless point-to-point connections in 3.1, but the netbox-ui plugin doesn't show it.

**Comment by dywilson73** [comment]
I need this too. 
I hope its just a config thing, but it is really important to us as we have a lot of sites with point to point links between remote buildings. 
Has anyone managed to get wireless links to display? or is it a new feature thing?
